src/utils.js

```diff
@@ -132,7 +132,28 @@ const getDuplicatedPackages = (options = {}) => {
     return cleanFromFalsePositives;
 };
 
+const getDedupLock = (root) => {
+    const lockFilePath = path.join(root, 'webpack-dedup.lock');
```

Comment:
I'd extract this to a `lockFilePath` parameter instead so it can be configured by the consumer as an option of the plugin (then you don't need `root` as a param here, though you'll still need it in `writeDedupLock` I guess..)

Reply:
Added lockFilePath and defaulting to 'root/webpack-dedup.lock'